portfolio/src/main/webapp/script.js

```diff
@@ -13,14 +13,14 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random quote or fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuoteOrFact() {
+  const quotesOrFacts =
+      ['Loves spicy food', 'Can speak some Korean', '"My Cabbages"', '"Bonjour le monde!"'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const greeting = quotesOrFacts[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
```

Comment:
I see that you filled in the method to show a random quote or fact on your page here, but didn't see you used it in your html file. Do you plan to use this method?

Reply:
I'm not sure. I suppose I could delete it.

Reply:
I am now trying to incorporate that code for a button.

Reply:
That makes sense. This PR looks good to me otherwise, you can merge it now so the commits don't show up in your second PR

Reply:
Last comment based on our discussion in Chat: Please change to quotesOrFacts.length